src/function.ts

```diff
@@ -42,34 +40,32 @@ export type AnyLambda = Function<AnyFunction>;
  *
  * Make sure to implement your Lambda Function entry-points accordingly.
  */
-export class Function<F extends AnyFunction> {
+export class Function<P = undefined, O = void> {
```

Comment:
Do we need default types? I think we should encourage the user to specify them. Also the defaults represent a pretty uselsss function haha

Reply:
is it useless?

`start()`
`record()`
`increment()`
`submit()`

But sure, we could get rid of the defaults.

Reply:
I guess not useless then. Useful for timers and alike. But probably not common?

Reply:
Removed the defaults, always can add it back later.